webapp/channels/build/emoji/make_emojis.mjs

```diff
@@ -54,12 +49,48 @@ const argv = yargs(process.argv.slice(1)).
 
 const argsExcludedEmojiFile = argv['excluded-emoji-file'];
 
+function log(level = '', text) {
+    if (level === 'info') {
+        // eslint-disable-next-line no-console
+        console.log(chalk.cyan(`[INFO]: ${text}`));
```

Comment:
@M-ZubairAhmed , just curious about console.log, should this be in production code?

Reply:
This is actually fine because this script is run before the app is run and that to in cases where we update the emojis in app.

Reply:
got it. 
I think if its not for the User , it shouldn't be pushed.

if you strongly feel this is needed, we can merge. 😄

Reply:
Yes this is fine because this is a script